example/rtld/profiles_waf/profiles_waf.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/auth"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/rtld"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/rtld/profiles_waf"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/rtldmodels"
+	"github.com/fatih/color"
+)
+
+func main() {
+
+	// Setup
+	apiToken := "MY_API_TOKEN"
+
+	idsCredentials := auth.OAuth2Credentials{
+		ClientID:     "CLIENT_ID",
+		ClientSecret: "CLIENT_SECRET",
+		Scope:        "SCOPE",
+	}
+
+	sdkConfig := edgecast.NewSDKConfig(apiToken, idsCredentials)
+
+	rtldService, err := rtld.New(sdkConfig)
+
+	if err != nil {
+		color.Red("error creating rtld Service: %v\n", err)
+		return
+	}
+
+	fmt.Println("")
+	color.Green("**** GET WAF CUSTOMER PROFILE ****")
+	fmt.Println("")
+
+	profileGetParam := profiles_waf.NewProfilesWafGetCustomerSettingsParams()
+	profileWafGetCustomerSettingsResp, err := rtldService.ProfilesWaf.ProfilesWafGetCustomerSettings(profileGetParam)
```

Comment:
Can we update to stay within 50 chars per line?

Reply:
Updated it.